CHANGELOG.md

```diff
@@ -7,20 +7,21 @@ This document contains a historical list of changes between releases. Only
 changes that impact end-user behavior are listed; changes to documentation or
 internal API changes are not present.
 
-Main (unreleased)
------------------
+## Main (unreleased)
 
-v0.30.1 (2022-12-23)
---------------------
+### Enhancements
```

Comment:
What happened with this file? Can you revert all the extra changes, please?

Reply:
looks like my markdown formatter. I'll fix this.